_src/app/templates/App.html

```diff
@@ -4,18 +4,13 @@
             <span class='title' data-dojo-attach-point='egg'>Atlas Utah</span>
             <a data-dojo-attach-point='version' class='version' href='ChangeLog.html'></a>
         </h1>
-        <img src="app/resources/agrc_logo.jpg" alt="agrc logo">
```

Comment:
i think double quotes are more common for html attributes but i'm not too worried about it.